lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -0,0 +1,101 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'js/financial-assistance/models/financial_assistance_model',
+            'text!js/financial-assistance/templates/financial_assessment_form.underscore',
+            'text!js/financial-assistance/templates/financial_assessment_submitted.underscore',
+            'js/student_account/views/FormView',
+            'text!templates/student_account/form_field.underscore'
+         ],
+         function(Backbone, $, _, gettext, FinancialAssistanceModel, formViewTpl, successTpl, FormView, formFieldTpl ) {
+         	return FormView.extend({
+         		el: '.financial-assistance-wrapper',
+         		events: {
+					'click .js-submit-form': 'submitForm'
+				},
+         		tpl: formViewTpl,
+         		fieldTpl: formFieldTpl,
+         		formType: 'financial-assistance',
+         		requiredStr: '',
+         		submitButton: '.js-submit-form',
+
+         		initialize: function(data) {
+					var context = data.context,
+						fields = context.fields;
+
+					// Add default option to array
+					if ( fields[0].options.length > 1 ) {
+						fields[0].options.unshift({
+							name: '- ' + gettext('Choose one') + ' -',
```

Comment:
This may have been a miscommunication. I saw in the sample options you hipchatted me that the 'Choose one' option wasn't included so I added it here. If you are adding it then I will remove this. Note that for the validation to work the following parameter also needs to be added ```default: true```